models/management.cattle.io.project.js

```diff
@@ -117,4 +117,15 @@ export default class Project extends HybridModel {
       return normanProject;
     })();
   }
+
+  // users with permissions for projectroletemplatebindings should be able to manage members on projects
+  get canUpdate() {
+    const canUpdateRoleBindings = this.$rootGetters['type-map/optionsFor']('management.cattle.io.projectroletemplatebindings');
+
+    return super.canUpdate || canUpdateRoleBindings.isEditable;
+  }
+
+  get canEditYaml() {
```

Comment:
This should have been brought in with master so shouldn't be needed (matches `super.canEditYaml`)